transcom/mymove
# INT B-18761 HL loading screen

## [B-18761](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18761)

## Summary

Ensures that the loading placeholder for the move history log stays in place until the table has actually been populated with data. One-line fix, added to if statement for LoadingPlaceholder to also render the placeholder if table data is <=0 and an error has not yet occured.

### How to test

1. Access the Move History tab of any move as any user role.
2. In Chrome devtools, go to the "Network" tab, on the top ribbon next to the wifi symbol, you should see a dropdown to enable network throttling, choose either fast or slow 3G. Slow will take awhile but the effect will be easier to see.
![Screenshot 2024-03-28 at 12 14 07 PM](https://github.com/transcom/mymove/assets/146959895/42679235-d10a-4f1d-b77f-f62444b9105e)

3. After enabling, reload the move history page, you should see that the loading placeholder stays in place right up until the table is shown, fully populated. Contrast this to the main/int branch, where doing the same process will result in an empty table showing on the screen for a few seconds until it is populated.
